bufr2geojson/__init__.py

```diff
@@ -50,36 +50,47 @@
 
 LOGGER = logging.getLogger(__name__)
 
-# some 'constants'
+# some 'constants' / env variables
 SUCCESS = True
 NUMBERS = (float, int, complex)
 MISSING = ("NA", "NaN", "NAN", "None")
-NULLIFY_INVALID = True  # TODO: move to env. variable
-
-BUFR_TABLE_VERSION = 37  # default BUFR table version
+NULLIFY_INVALID = os.environ.get("BUFR2GEOJSON_NULLIFY_INVALID", True)
 THISDIR = os.path.dirname(os.path.realpath(__file__))
 RESOURCES = f"{THISDIR}{os.sep}resources"
 CODETABLES = {}
-
+FLAGTABLES = {}
 ECCODES_DEFINITION_PATH = codes_definition_path()
 if not os.path.exists(ECCODES_DEFINITION_PATH):
     LOGGER.debug('ecCodes definition path does not exist, trying environment')
     ECCODES_DEFINITION_PATH = os.environ.get('ECCODES_DEFINITION_PATH')
     LOGGER.debug(f'ECCODES_DEFINITION_PATH: {ECCODES_DEFINITION_PATH}')
     if ECCODES_DEFINITION_PATH is None:
         raise EnvironmentError('Cannot find ecCodes definition path')
-
 TABLEDIR = Path(ECCODES_DEFINITION_PATH) / 'bufr' / 'tables' / '0' / 'wmo'
 
+# ToDo - read preferred units from config file
 # PREFERRED UNITS
 PREFERRED_UNITS = {
     "K": "Celsius",
     "Pa": "hPa"
 }
 
+# The following is required as the code table from ECMWF is incomplete
+# and that from github/wmo-im not very usable.
+try:
+    with open(f"{RESOURCES}{os.sep}031021.json") as fh:
```

Comment:
perhaps define/reference this filename at the of this file for easy updates in the future?